### PR TITLE
feat(tutorial): complete step 5

### PR DESCRIPTION
**Pull request by fennygai**

### CODE REVIEW

File: src/styles/_carbon.scss [hunk 1]
```diff
@@ -2,5 +2,21 @@ $feature-flags: (
   grid-columns-16: true
 );
 
-@import 'carbon-components/scss/globals/scss/styles';
+// @import 'carbon-components/scss/globals/scss/styles';
 @import './carbon-fixes';
```
Comment (lee-chase) [inline src/styles/_carbon.scss]: The carbon fixes need to come after the other carbon imports.

Comment (fennygai) [reply src/styles/_carbon.scss]: > The carbon fixes need to come after the other carbon imports.

thanks for the review, have change this in another commit

**Comment by fennygai** [comment]
IBM Cloud deployment  linke is :   https://carbon-tutorial-vue-fennygai.mybluemix.net/#/

**Comment by fennygai** [comment]
@lee-chase could you please help to review the pr again, thanks!

**Comment by lee-chase** [comment]
:thumbsup::100::clap::smile::star:fantastic, you have completed the tutorial and if you have completed all five steps you can now take a look at the wrapping up step where you will find instructions on claiming the Carbon Developer Essentials Vue badge. Feel free to stop your running application on IBM Cloud to avoid the possibility of incurring any charges.

**Pull request closed without merging**